preproc/Threshold_calibration_summary.ipynb

```diff
@@ -340,10 +330,35 @@
     "df_data.tofpet_id.unique()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "c1eea4a4",
+   "metadata": {},
+   "source": [
+    "We use both plots shown below to verify if the data collection process has gone fine.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "01952b6f",
+   "metadata": {},
+   "source": [
+    "In the first plot we show the data entries in time. There is no a fix number of entries, as it depends on\n",
```

Comment:
Remember: `fix` is not an adjective. You mean `fixed number of entries`, where `fixed` is the past participle.

Reply:
done!